packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -430,32 +436,29 @@ export const BlockBindingsPanel = ( { name: blockName, metadata } ) => {
 				bindableAttributes: _bindableAttributes,
 			};
 		},
-		[ blockContext, blockName ]
-	);
-	// Return early if there are no bindable attributes.
+		[ blockContext, blockName, metadata ]
+	); // Return early if there are no bindable attributes.
```

Comment:
The comment refers to the line below -- IMO it should not be next to the closing parenthesis.

```suggestion
	);
	// Return early if there are no bindable attributes.
```

Reply:
Fixed.